src/app.css

```diff
@@ -0,0 +1,118 @@
+/* GLOBAL STYLES */
+
+/* montserrat-alternates-regular - latin */
```

Comment:
In mijn ticket zet ik deze nog even om naar een `fonts.css`, want anders is het echt onoverzichtelijk. Voor nu is dit prima hoor. Heeft geen zin om dat hier ook aan te passen. Laat die merge conflicten maar bij mij komen 😅